types/trigger.d.ts

```diff
@@ -190,7 +190,7 @@ export type BaseTriggerSet<Data extends RaidbossData> = {
 };
 
 // If Data contains required properties that are not on RaidbossData, require initData
-export type TriggerSet<Data extends RaidbossData> =
+export type TriggerSet<Data extends RaidbossData = RaidbossData> =
```

Comment:
What does this change do? Sorry for my lack of knowledge, I just am not sure what `= RaidbossData` does.

Reply:
default generic argument

Reply:
Thanks for the explanation.  It's just hard to search for something like that.  I didn't know you could do defaults on generics.